docs/.vuepress/theme/components/Home.vue

```diff
@@ -73,6 +73,10 @@ export default {
               title: 'Connect to a network',
```

Comment:
As you suggested on Slack, please suggest a new title for this page to disambiguate